create-update.sh

```diff
@@ -130,6 +133,10 @@ if [ $NOPUBLISH -eq 0 ]; then
 	sudo cp "$PWD/.mix-version" "$STATE_DIR/www/version/format$FORMAT/latest"
 fi
 
-# step 6: archive the swupd-server logs for this mix build
+# step 6: create delta packs and super packs
+sudo mixer-pack-maker.sh $MIXVER $BACK_COUNT
```

Comment:
This script should accept a $STATE_DIR arg as well, since it may be run from any location w/arbitrary state_dir.